Pod/Classes/BONText.h

```diff
@@ -88,6 +88,11 @@ typedef NS_ENUM(NSUInteger, BONFigureSpacing) {
 @property (copy, nonatomic, readonly, BONNonnull) NSAttributedString *attributedString;
 @property (copy, nonatomic, readonly, BONNonnull) BONStringDict *attributes;
 
+/**
+ *  @c YES if and only if the resulting @c attributedString will be the empty attributed string.
+ */
+@property (nonatomic, readonly) BOOL empty;
```

Comment:
`empty` seems like a weird name for this property as I personally wouldn't consider a `BONText` empty if I'd assigned any of the properties. This feels like it should be named something more like `hasString`, though as discussed that is somewhat confusing and might be interpreted as `string != nil`.